src/auth/cephx/CephxKeyServer.h

```diff
@@ -30,6 +30,10 @@ struct KeyServerData {
   /* for each service type */
   version_t rotating_ver;
   std::map<uint32_t, RotatingSecrets> rotating_secrets;
+  KeyServerData() 
+    : version(0), 
```

Comment:
nit: we can initialize at member's definition, e.g.:

```cpp
version_t version{0};
```

Reply:
Wanted to follow the exist code, but yes, i'll change that